srcpkgs/salt/template

```diff
@@ -1,21 +1,20 @@
 # Template file for 'salt'
 pkgname=salt
-version=3000.2
+version=3001.1
 revision=1
 archs=noarch
-build_style=python2-module
-pycompile_module="salt"
-hostmakedepends="python-devel python-setuptools"
-depends="python-yaml python-Jinja2 python-requests python-pyzmq
- python-M2Crypto python-tornado python-msgpack dmidecode pciutils
- python-psutil"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-yaml python3-Jinja2 python3-requests python3-pyzmq
+ python3-M2Crypto python3-tornado python3-msgpack dmidecode pciutils
+ python3-psutil python3-distro"
```

Comment:
Should `python3-pycryptodomex` be added here?

Reply:
🤦‍♂️ Dunno how I missed that.